docs/flexbox.md

```diff
@@ -33,11 +60,27 @@ export default class FlexDirectionBasics extends Component {
 // skip this line if using Create React Native App
 AppRegistry.registerComponent('AwesomeProject', () => FlexDirectionBasics);
 ```
+See some examples of [wix](https://medium.com/wix-engineering/the-full-react-native-layout-cheat-sheet-a4147802405c) engineers
+![Flex Direction](https://cdn-images-1.medium.com/max/800/1*rA7IbuUsJWsx6evKAsabVw.png)
 
 #### Justify Content
 
+[`justifyContent`](https://facebook.github.io/react-native/docs/layout-props#justifycontent) describes how to align children within the main axis of their container. For example, you can use this property to center a child horizontally within a container with `flexDirection` set to `row` or vertically within a container with `flexDirection` set to `column`.
+
+- `flex-start`(**default value**) Align children of a container to the start of the container's main axis.
+
+- `flex-end` Align children of a container to the end of the container's main axis.
+
+- `center` Align children of a container in the center of the container's main axis.
+
+- `space-between` Evenly space of children across the container's main axis, distributing remaining space between the children.
+
+- `space-around` Evenly space of children across the container's main axis, distributing remaining space around the children. Compared to `space between` using `space around` will result in space being distributed to the beginning of the first child and end of the last child.
+
 Adding `justifyContent` to a component's style determines the **distribution** of children along the **primary axis**. Should children be distributed at the start, the center, the end, or spaced evenly? Available options are `flex-start`, `center`, `flex-end`, `space-around`, `space-between` and `space-evenly`.
```

Comment:
This paragraph is now just repeating what's stated right before it, should we remove it?

Reply:
Let's make sure to keep the yoga link

Reply:
Excellent!